src/main/java/com/exadel/discountwebapp/exception/WebExceptionHandler.java

```diff
@@ -13,6 +14,7 @@
 
 import static org.springframework.http.HttpStatus.*;
 
+@Log4j2
```

Comment:
The default logger implementation used by Spring Boot is Logback, not Log4j2. So, let's use Slf4j annotation

Reply:
replaced